screen-pop-app/screen-pop/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn start`
```

Comment:
Don't change anything here, but for future reference, use npm when setting up new react apps instead of yarn. Yarn had its place at one point, but there's no good reason to use it anymore for new projects. NPM tends to be less finicky these days.

Reply:
got it